.github/workflows/code_quality.yml

```diff
@@ -0,0 +1,31 @@
+name: 🚧 Run code quality checks
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
```

Comment:
@TangoYankee @horatiorosa Maybe this is something we visit but would it be difficult to move these code quality checks into the workflows for deploying the environments? I much prefer things like code quality and tests being in the same workflow as deployments, as separate "jobs". That way you have the option of blocking the deployment if the code quality checks fail. We use this approach in [EDDE's workflows](https://github.com/NYCPlanning/equity-tool/blob/main/.github/workflows/feature_branch.yml).

Reply:
yes and no to the difficulty question. We can just copy and paste the code quality checks into each environment workflow.

Long term, we should make the jobs reusable, otherwise we end up needing to update code across several workflows every time we make an updated. Of course, this goes for code quality and deployment jobs (we're mostly reusing code across the deployment workflows, too)

For now, would you like to go forward with copying and pasting code quality checks into each deployment workflow?

Reply:
Yes please.